src/types.ts

```diff
@@ -1173,6 +1174,7 @@ export type AppSettings = {
     notification_template?: string;
     server_key?: string;
   };
+  image_moderation_enabled?: boolean;
```

Comment:
I'm not so sure about this, because it seems to be missing most app setting....

Reply:
I followed the test code:

```js
                await client.updateAppSettings({
			image_moderation_enabled: true,
		});
```
And the method description is:

```js
async updateAppSettings(options: AppSettings)
```

I'm not sure how this tests are performed, but it seemed logical to put this option there too

Reply:
Will try to run tests without this field